assets/css/style.scss

```
@@ -407,6 +407,22 @@ h2.gala-heading {
     color: $white;
 }
 
+.sponsorship-image {
+  background-image:url(https://res.cloudinary.com/zaigezaigu/image/upload/v1670371900/zgzg-io-website/%E8%B5%9E%E5%8A%A9/jason-goodman-wVh5grSMYaY-unsplash_bupoj8.jpg);
+  background-size: cover;
+  background-repeat:no-repeat;
+  margin: 0 auto 0 auto;
+  min-width: 100%;
+  padding-top:50px;
```

Format your code properly.
```suggestion
  background-repeat: no-repeat;
  margin: 0 auto 0 auto;
  min-width: 100%;
  padding-top: 50px;
```